modules/generating-icsp-object-scoped-to-a-registry.adoc

```diff
@@ -0,0 +1,61 @@
+[id="generating-icsp-object-scoped-to-a-registry_{context}"]
+= Widening the mirror registry scope of the ImageContentSourcePolicy object
+
+Use the `--icsp-scope=registry` flag with the `oc adm catalog mirror` command to widen the mirror registry scope in the `ImageContentSourcePolicy` object. This minimizes the number of times the cluster nodes are rebooted when a registry entry is updated.
```

Comment:
Passive voice. Perhaps "This minimizes the number of times the cluster nodes reboot when updating a registry entry."